src/Integration.Vsix/Helpers/FileUtilities.cs

```diff
@@ -0,0 +1,61 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace SonarLint.VisualStudio.Integration.Vsix.Helpers
+{
+    internal static class FileUtilities
+    {
+        [DllImport("shlwapi.dll", CharSet = CharSet.Auto)]
+        private static extern bool PathRelativePathTo(
```

Comment:
No need for P/Invoke here. I've run a quick test and this method passes the tests, so I'd prefer we use it instead:
https://stackoverflow.com/a/340454

Reply:
Discussed offline: portability isn't an issue here, and P/Invoke call does the job in one line rather multiple., so I'd rather keep the P/Invoke version.